.github/workflows/pr-windows-ae.yml

```diff
@@ -119,17 +121,27 @@ jobs:
       with:
         name: dist
         path: ${{github.workspace}}/build/dist
-
     - uses: actions/upload-artifact@v3
       with:
         name: coverage
         path: ${{github.workspace}}/build/__coverage
 
+      # - run: |
+      #     New-Item -ItemType Directory -Path ${{github.workspace}}/build/nexus/sp/data/ -Force
+      #     New-Item ${{github.workspace}}/build/nexus/sp/data/foo.bar.txt
     - uses: actions/upload-artifact@v3
       with:
         name: SP-AE
         # Data folder is skipped for mod managers
         path: ${{github.workspace}}/build/nexus/sp/data/*
+    # create a zip archive
+    - run: |
+        $zip = ${{github.workspace}}/build/nexus/sp.zip
+        $source = ${{github.workspace}}/build/nexus/sp/data
+        Compress-Archive -Path $source -DestinationPath $zip
+    - run: |
+        dotnet tool install -g Digitalroot.ModUploader
+        drmu nexusmods upload 54909 ${{github.workspace}}/build/nexus/sp/data --file-name "Skyrim Platform 2.7.0 (Anniversary Edition)" --version "2.7.0" --category Main --description "THIS IS FOR ANNIVERSARY EDITION (1.6.x)!\n\nPlease backup your saves before updating. There are issues with some modlists.\n\nChangelog: https://github.com/skyrim-multiplayer/skymp/blob/main/docs/release/sp-2.7.0.md\n\nFor questions: https://discord.gg/k39uQ9Yudt" --game skyrimspecialedition
```

Comment:
Can the `2.7.0` be turned into a variable that can be changed in one place in the future?

Reply:
I was thinking about leaving 2.7.0 here and adding this file to https://github.com/skyrim-multiplayer/skymp/blob/main/.github/workflows/sp-release.yml#L29

but yeah well maybe I just could read the version instead